Makefile

```diff
@@ -45,6 +45,6 @@ build-example-pretty:
 docs:
 	jazzy
 
-dist: clean-pretty format test-pretty build-pretty docs lint
+dist: clean-pretty test-pretty build-pretty lint jazzy
```

Comment:
small thing - are we intending to not run `format` going forward?

Reply:
Good question. When I run it, it does weird things to the indentation; when Nick runs it, it's fine. He's using an older version, I think. Maybe I can roll mine back using Homebrew. Whatever the case, it directly conflicts with Xcode's own formatting rules (ctrl+I). I can run it and then manually undo some of the weird changes.

Reply:
Got it, I think it's fine to remove it for now given that the latest version of format isn't doing what we want.  I'll add something to the SDK roadmap to figure out what we want to do here going forward.

Reply:
For this release, I recommend just skipping it.